VerticalSlice2D/Assets/Scripts/Heart/HeartCollisions.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HeartCollisions : MonoBehaviour
+{
+    private HeartInvisibility heartInvisibility;
+
+    private void Start()
+    {
+        heartInvisibility = GetComponent<HeartInvisibility>();
+    }
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        if (collision.transform.tag == "Ball")
```

Comment:
Het is volgens mij beter om _if (transform.CompareTag("Ball"))_ te gebruiken want dat is iets netter en efficienter

Reply:
Thanks voor de tip! Het is aangepast